### ISSUE TITLE
Files are not removed on pressing 'Remove'

### ISSUE THREAD

**Issue by int-ua** [bradleyg/django-ajaximage#19]
I've tried to upload an image three times and now there are three copies of it in the media folder.

**Comment by bradleyg** [comment]
Yes, I guess this is confusing. Files are not actually removed from the filesystem, but the reference is removed from the model.

We should delete the file as well.
